SCA_DB_script.sql

```diff
@@ -50,33 +50,33 @@ CREATE TABLE dbo.[Assailant] (
 GO
 
 CREATE TABLE [Area] (
-  [Area_ID] [int] IDENTITY(1,1) PRIMARY KEY NOT NULL,
-  [ZIP_Code] [int],
-  [Suburb] [varchar](150),
-  [City] [varchar](150) ,
-  [Province] [varchar](150),
+  [Area_ID] [INT] IDENTITY(1,1) PRIMARY KEY NOT NULL,
+  [ZIP_Code] [INT] NOT NULL,
+  [Suburb] [varchar](150) NOT NULL,
+  [City] [varchar](150) NOT NULL,
+  [Province] [varchar](150) NOT NULL,
 );
 GO
 
 
 CREATE TABLE [Location] (
-  [Location_ID] [int] IDENTITY(1,1) PRIMARY KEY NOT NULL,
-  [Area_Code_ID] [int] FOREIGN KEY REFERENCES [Area]([Area_ID]) NOT NULL,
-  [Street_Number] [int],
-  [Street_Name] [varchar](250),
-  [Latitude] [decimal](10,6),
-  [Longitude] [decimal](10,6),
+  [Location_ID] [INT] IDENTITY(1,1) PRIMARY KEY NOT NULL,
+  [Area_Code_ID] [INT] FOREIGN KEY REFERENCES [Area]([Area_ID]) NOT NULL,
+  [Street_Number] [INT] NULL,
+  [Street_Name] [varchar](250) NULL,
+  [Latitude] [decimal](10,6) NOT NULL,
+  [Longitude] [decimal](10,6) NOT NULL,
 );
 GO
 
 
 CREATE TABLE dbo.[Incident] (
-  [Incident_ID] INT IDENTITY (1,1) NOT NULL PRIMARY KEY,
-  [User_ID] INT NOT NULL,
-  [Location_ID] INT NOT NULL,
-  [Date] DATE NOT NULL,
-  [Time] TIME NOT NULL,
-  [Description] varchar(150) NOT NULL,
+  [Incident_ID] [INT] IDENTITY (1,1) NOT NULL PRIMARY KEY,
+  [User_ID] [INT] NOT NULL,
+  [Location_ID] [INT] NOT NULL,
+  [Date] [DATE] NOT NULL,
+  [Time] [TIME] NOT NULL,
+  [Description] varchar(150) NULL,
```

Comment:
Should this varchar not also be in square brackets like the rest?

Reply:
It should! Thank you :)

Reply:
Pleasure, sorry didn't see this reply. Also sent a message on teams.